src/components/InputTitle.tsx

```diff
@@ -0,0 +1,33 @@
+import { css } from '@emotion/react';
+
+interface InputTitlePropsType {
+  title: string;
+  subtitle?: string;
+}
+
+function InputTitle({ title, subtitle }: InputTitlePropsType) {
+  return (
+    <div css={titleContainerStyle}>
+      <h1 css={titleStyle}>{title}</h1>
+      <p css={subtitleStyle}>{subtitle}</p>
```

Comment:
subTitle 은 옵셔널 타입이니까 조건부 렌더링 처리 해줘야하지 않을까요?
```suggestion
      {subtitle && <p css={subtitleStyle}>{subtitle}</p>}
```